src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -162,30 +158,10 @@ def update_argument(self, param_name, argtype):
         arg.type.update(other=argtype)
 
     def __call__(self, *args, **kwargs):
-
-        from azure.cli.core import AzCommandsLoader
-        cmd_args = args[0]
-
         if self.command_source and isinstance(self.command_source, ExtensionCommandSource) and\
            self.command_source.overrides_command:
             logger.warning(self.command_source.get_command_warn_msg())
-        if self.deprecate_info is not None:
```

Comment:
I'm assuming then that deprecate info is part of knack?

Reply:
Yep https://github.com/Microsoft/knack/blob/master/knack/commands.py#L97